packages/zipkin/src/tracer/index.js

```diff
@@ -37,13 +39,15 @@ class Tracer {
     localEndpoint,
     /* eslint-disable no-console */
     log = console,
-    defaultTags
+    defaultTags,
+    propagation = new B3Propagation()
   }) {
     this.log = log;
     this.recorder = recorder;
     this.sampler = sampler;
     this.traceId128Bit = traceId128Bit;
     this.supportsJoin = supportsJoin;
+    this._propagation = propagation;
```

Comment:
Don't know if a better approach is delete the extract and injector methods and only add a get propagation, this simplify the grpc-client implementation, thats need to know the propagation headers....

Reply:
I would go for that instead.